dt-backend/src/app.module.ts

```diff
@@ -12,6 +19,18 @@ import { AuditLogModule } from './model/audit-log/audit-log.module';
       autoSchemaFile: `${process.cwd() + '/src/schema.gql'}`,
     }),
     AuditLogModule,
+
```

Comment:
Same here - those empty lines just make the file unnecessary long! Please avoid them!  
Side Note: the `UserModule` is at the top, so feel free to either take everything to the top or everything to the bottom! Just don't create empty spaces!